### ISSUE TITLE
"only check current editor" does not do what it says

### ISSUE THREAD

**Issue by travnick** [CJCombrink/SpellChecker-Plugin#101]
There is an option "only check current editor" but it does not do what it says.

It simply means "do spellcheck in current (edited) file only"
Unfortunately there is additional hidden condition "in active project"

It would be nice to have this renamed - to properly describe the action and conditions:
"check only active project". Or choose the most appropriate words that describe the behavior.

(there is a documentation - but the best documentation is meaningful name, that does not need any explanation)

**Comment by CJCombrink** [comment]
I agree, this is something that is also irritating me, especially when I work on libraries. 
I would like it to check the current edited file in any project, not just the active one, that would be the goal. 

At one point I did try to resolve this and can't remember why I abandoned the effort. It might have had something to do with it trying to spellcheck library files (Qt, std, boost, etc) when I opened them which degraded the experience a lot.

I will look at this again when I get a chance (Unfortunately I can't work on the SpellChecker for the next few weeks)

**Comment by CJCombrink** [comment]
Note-to-self: Look into how Qt Creator adds the warnings on top of the editor when something goes wrong to indicate when a file was not spell checked.

Existing examples are: "Note: Multiple parse contexts are available for this file..." and "Warning: The code model could not parse the included file..."

**Comment by travnick** [comment]
What do you mean by `I would like it to check the current edited file in any project, not just the active one`
What is the difference between those two?
